src/main/java/mate/academy/model/User.java

```diff
@@ -0,0 +1,66 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "users")
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String login;
```

Comment:
Let's make the email unique. We need to check the email during user registration and make the email a unique field in the database.
```suggestion
    private String email;
```

Reply:
Ok